tests/features/java/openjdk_s2i.feature

```diff
@@ -2,6 +2,7 @@
 @ubi8/openjdk-11
 @ubi8/openjdk-17
 Feature: Openshift OpenJDK-only S2I tests
+
   @ubi8/openjdk-8
   Scenario: Check java perf dir owned by jboss (CLOUD-2070)
```

Comment:
Are you sure this is the gist of the patch? If so, please explain why that works?

Reply:
The 'Feature' line has the other image names tagged above it. Other Scenarios later in the file (separated from both the Feature and Scenario lines by whitespace) inherit their tagging from the Feature. 

Without this newline, the 'Feature' line, above, and the 'Scenario' line, below, are considered part of the same stanza, and so the tag `@ubi8/openjdk-8` is applying to the Feature as well, and being inherited by all the other Scenarios (including the CDS one).

Reply:
Oh boy, what terrible design. Thanks for the explanation.